_posts/2019-03-05-and-then-there-was-one-metadata-links.md

```diff
@@ -0,0 +1,24 @@
+---
+title: And then there was one
```

Comment:
Let's use a more descriptive title

Reply:
updated to:
> Removing the edit linkset form

Reply:
a more descriptive but rather wordy one could be:
> Removing the edit linkset form and introducing metadata URI attributes.